models/receipt.go

```diff
@@ -111,13 +111,19 @@ func MarshalReceipt(
 		return map[string]interface{}{}, err
 	}
 
+	var to *string
```

Comment:
What about (after we define fields):
```
	if tx.To() != nil {
		fields["to"] = tx.To().Hex()
	}
	```

Reply:
Nice :+1: That's much better!

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/195/commits/698a5d90bac0ec11072ac464f115cff3aa17eb4e